tools/walletextension/common/common.go

```diff
@@ -83,3 +86,36 @@
 	}
 	return encClient, nil
 }
+
+type RPCRequest struct {
+	ID     json.RawMessage
+	Method string
+	Params []interface{}
+}
+
+// Clone returns a new instance of the *RPCRequest
+func (r *RPCRequest) Clone() *RPCRequest {
+	return &RPCRequest{
+		ID:     r.ID,
+		Method: r.Method,
+		Params: r.Params,
+	}
+}
+
+// Formats the log to be sent as an Eth JSON-RPC response.
```

Comment:
Doesnt's seem to be used other than the subscription package ?

Reply:
currently it is still used in `executeSubscribe` in `account_manager.go`, but once that method is removed I will move this code to subscripton package (adding a todo now)